io.github.torbuntu.Leikr.json

```diff
@@ -0,0 +1,42 @@
+{
+    "id": "io.github.torbuntu.Leikr",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "command": "/app/Leikr/Leikr",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk"],
+    "finish-args": [
+        "--env=PATH=/app/jre/bin:/usr/bin",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-config",
+        "--device=all"
+    ],
+    "modules": [{
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": ["/usr/lib/sdk/openjdk/install.sh"]
+        }, {
+            "name": "leikr",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install --directory --mode=0755 /app/Leikr/",
```

Comment:
Why 755 permission?

Reply:
I saw this was done in [Intellij](https://github.com/flathub/com.jetbrains.IntelliJ-IDEA-Community/blob/master/com.jetbrains.IntelliJ-IDEA-Community.json) and may have misunderstood the use for it.

Reply:
The executable is stored in that directory (see "command").